src/single.ts

```diff
@@ -121,6 +122,10 @@ export class RegionRatelimit extends Ratelimit<RegionContext> {
      * The duration in which `tokens` requests are allowed.
      */
     window: Duration,
+    /**
+     * How many custom rates(if any) are allowed per window.
+     */
+    rates?: number,
```

Comment:
`rates` feels so generic tbh. Let's change it to something like `customLimit`.

Reply:
Yes it's too generic and confusing for me as well. Anyways I will change that